.github/workflows/merge-prometheus-operator.yaml

```diff
@@ -0,0 +1,20 @@
+name: Prometheus Operator merger
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' #@daily
+jobs:
+  prometheus-operator-merge:
+    uses: rhobs/syncbot/.github/workflows/merge-flow.yaml@main
+    with:
+      upstream: prometheus-operator/prometheus-operator
+      downstream: openshift/prometheus-operator
+      sandbox: rhobs/prometheus-operator
+      go-version: 1.17
+      restore-upstream: CHANGELOG.md VERSION go.mod go.sum
```

Comment:
nit:  may want to consider using `>-` and sort the files for better readabililty 
e.g the following produces the same output
```yaml
restore-upstream: >-
   CHANGELOG.md
   VERSION
   go.mod
   go.sum
```

Reply:
Thanks @sthaha . Updated according to your suggestion.